### DISCUSSION TITLE
Can't enable laser mode

### DISCUSSION THREAD

**Discussion by rjthemxguy** [arkypita/LaserGRBL#1982] [General]
I have a 3018 and when I send any laser command in GRBL, like $32=1, $32=0, etc, to the unit I get a red circle with the white line in it.

No fan, no laser.

Jogging works fine.  Just no laser commands.

Any ideas???

**Comment by Tribo-2dot0** [comment]
Qual versão do grbl você tem instalado?

**Comment by StuartB4** [comment]
Translation of Tribo-2dot0's question. 
What version of grbl do you have installed?

Also. 
Where are you entering it, where does the red circle appear, can you post a screen shot. 
Never seen it appear anywhere before. 

You should just be able to click Connect in LaserGRBL, Goto Grbl Configuration on the Grbl menu, change $32 to 1 and click the Write button to store the setting, then click Close.

**Comment by rjthemxguy** [comment]
Thanks for the response.  I don't have a $32 in my config.  It goes to $27
then jumps to the hundreds.

Weird.

**Comment by StuartB4** [comment]
Thats probably because you have an old version of GRBL. You need to update it. You can do that on the Tools menu of LaserGRBL. 
What does it say here? This is the latest version.

![yuyuyu](https://user-images.githubusercontent.com/16980026/209439218-d6dc2e66-c1d5-4bd0-9868-64fbadaad297.png)

**Comment by StuartB4** [comment]
If you are not sure how to update grbl you can follow these images. 

If you have a controller with the Arduino chip built in to the main board choose Arduino Nano from the drop down menu where it says Target.
If you have a seperate Arduino Uno board and use a CNC shield on the top of it then select Arduino Uno from the drop down menu. 

Select the Firmware from the next dropdown menu. If you don't have homing switches fitted to your engraver then just select the
1.1h original option.  


![Recovered_jpg_file(68)](https://user-images.githubusercontent.com/16980026/209441596-aa71597c-d634-45f7-8f44-ad9f646ddfa8.jpg)
![Recovered_jpg_file(69)](https://user-images.githubusercontent.com/16980026/209441597-afa00c9f-179a-4669-8dee-6d652c10f0e4.jpg)
![Recovered_jpg_file(70)](https://user-images.githubusercontent.com/16980026/209441600-27040d03-76fc-451e-a4e1-4ed2905dc21d.jpg)
![Recovered_jpg_file(71)](https://user-images.githubusercontent.com/16980026/209441601-cee15c2a-0f48-4404-a2b6-d54ba44c0231.jpg)
![Recovered_jpg_file(72)](https://user-images.githubusercontent.com/16980026/209441603-3c4de636-bc16-4802-959a-868a5886f007.jpg)
![Recovered_jpg_file(73)](https://user-images.githubusercontent.com/16980026/209441604-2ab8c1be-b61b-436e-b030-8f38e0201da4.jpg)
![Recovered_jpg_file(74)](https://user-images.githubusercontent.com/16980026/209441605-40b98146-302b-4d6e-9932-8ddcb6cfab8c.jpg)

**Comment by StuartB4** [comment]
Then do this. Start LaserGRBL and click Connect. 
Open GRBL Configuration. 


![iopiiipo](https://user-images.githubusercontent.com/16980026/209442020-56673687-2f53-4ada-a08d-1611926433dd.png)

**Comment by LsrSal** [comment]
rjthemxguy,

First of all, is your serial port connected? After contention - have you got UART post with GRBL version, like in message from StuartB4?

Then, are you in alarm mode? You will not be able to write anything till alarm is cleared. You may type "$X" to clear it, assuming serial is working.
